test/suites/projects.sh

```diff
@@ -321,7 +321,10 @@ test_projects_profiles() {
   # Try project copy
   lxc project create foo
   lxc profile copy --project default --target-project foo default bar
+  lxc profile copy --project default --target-project foo default bar --refresh
```

Comment:
To not only check if the command succeeds in general, you could add an additional test for the values itself:

```bash
lxc profile set --project default default user.x z
lxc profile copy --project default --target-project foo default bar --refresh
lxc profile get --project default default user.x | grep -q 'z'
```

Reply:
ok tests has been updated. I also added a case when we copy to an existing profile without --refresh, which should fail.

Reply:
Thanks!